src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/Microsoft.DotNet.UpgradeAssistant.Steps.Packages.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
does it make sense to just move to 6.0 here?

Reply:
There aren't APIs there we may want to use